Proposed new training

GiudGiud opened the pull request:
- [x] Make all steps work
- [x] add more professional context, with INL graphics and co. Make it more promotional
- [x] add pre-checked in graphics
- [x] remove more C++ from systems explanation

This will have to wait for  https://github.com/idaholab/moose/pull/29072
Doco test failure will be addressed by #29729 

There are a few other developments required in there. Mostly bug fixes and a little bit of new capability

what could we do more?
- add more content in syntax / systems descriptions
- add more examples
- add more steps
  - a workbench step
  - a meshing step
 
Would be good to do this later.

GiudGiud committed:
Update general tutorial index for new trainings, refs #21242

GiudGiud committed:
Add config files for low dev workshop

GiudGiud committed:
Add diffusion step 1 on shield

GiudGiud committed:
Add common training content (duplicated for now)

GiudGiud committed:
Add diffusion + a coef step 2

GiudGiud committed:
Add step3: boundary conditions

GiudGiud committed:
Hope this fixes the website

GiudGiud committed:
Rename files to avoid name conflict

GiudGiud committed:
Add step 4: material properties for kernels

GiudGiud committed:
Add step 5: auxiliary variables

GiudGiud committed:
Add step 6: transient simulation

GiudGiud committed:
Add step 7: coupling with fluid flow

GiudGiud committed:
Add step 8: adaptivity
no videos, we ll have to generate them ourselves

GiudGiud committed:
Add step 9 : postprocessing
 no videos, we ll have to generate them ourselves

GiudGiud committed:
Add step 10: thermo mechanics

GiudGiud committed:
Add summaries

GiudGiud committed:
Use 1st order NSFE flow with stab

GiudGiud committed:
Add step11: multiapps

GiudGiud committed:
Add step 12: action syntax

GiudGiud committed:
misc change

GiudGiud committed:
MOre work on CHT

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2217847/) on aff2e3c wanted to post the following:

Warning: This PR changes repo size by 22.59 MiB.

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/2217843/) on aff2e3c wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/27572/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/2217887/) on aff2e3c wanted to post the following:


## Framework coverage

<table>
  <thead>
    <tr>
      <th></th>
      <th>b3efaa</th>
      <th colspan=4>#27572 aff2e3</th>
    </tr>
    <tr>
      <th></th>
      <th>Total</th>
      <th>Total</th>
      <th>+/-</th>
      <th>New</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>Rate</b></td>
      <td>85.11%</td>
      <td>85.11%</td>
      <td>+0.00%</td>
      <td>-</td>
    </tr>
    <tr>
      <td><b>Hits</b></td>
      <td>103108</td>
      <td>103109</td>
      <td>+1</td>
      <td>0</td>
    </tr>
    <tr>
      <td><b>Misses</b></td>
      <td>18044</td>
      <td>18043</td>
      <td>-1</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

[Diff coverage report](https://mooseframework.inl.gov/docs/PRs/27572/coverage_diff/framework)

[Full coverage report](https://mooseframework.inl.gov/docs/PRs/27572/coverage/framework)


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/contact)
- [`electromagnetics`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/electromagnetics)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/geochemistry)
- [`heat_transfer`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/heat_transfer)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/navier_stokes)
- [`optimization`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/optimization)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/richards)
- [`scalar_transport`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/scalar_transport)
- [`solid_mechanics`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/solid_mechanics)
- [`solid_properties`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/solid_properties)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/stochastic_tools)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/xfem)

</details>



This comment will be updated on new commits.

GiudGiud commented:
not yet

GiudGiud committed:
Update syntax

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2633521/), step Size check on e233e0f wanted to post the following:

Warning: This PR changes repo size by 22.59 MiB.

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/2633517/), step Docs: sync website on e233e0f wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/27572/site)

This comment will be updated on new commits.

GiudGiud committed:
Update the meshing page

GiudGiud committed:
Add figure for multiapps
Improve auxkernel doco

GiudGiud committed:
Add intermediary steps back to the simulations
Add CI theory
Clean up more formatting

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2638369/), step Size check on 5b90683 wanted to post the following:

Warning: This PR changes repo size by 25.74 MiB.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/2638412/), step Generate coverage on 5b90683 wanted to post the following:


## Framework coverage

Coverage did not change


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/contact)
- [`electromagnetics`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/electromagnetics)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/geochemistry)
- [`heat_transfer`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/heat_transfer)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/navier_stokes)
- [`optimization`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/optimization)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/richards)
- [`scalar_transport`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/scalar_transport)
- [`solid_mechanics`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/solid_mechanics)
- [`solid_properties`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/solid_properties)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/stochastic_tools)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/27572/coverage/xfem)

</details>



This comment will be updated on new commits.

GiudGiud committed:
Improve training configs to be able to pull content from the other training

GiudGiud committed:
Rework user training index to point to local files

Adapt to folder re-organization

GiudGiud committed:
Fixups in regular tutorial
- add some text on CI

GiudGiud committed:
Prevent naming conflicts in regular tutorial

GiudGiud committed:
Set up testing of new tutorial
Use RunApp for now

GiudGiud committed:
Add a restart/recover hands on step

GiudGiud committed:
Remove all C++ from the tutorial problem steps

GiudGiud committed:
Add gravity to mechanics
Have a little more refinement in step 8 w/ adaptivity

GiudGiud committed:
Revamp step12 to use Physics

GiudGiud committed:
Let the DumpObjectProblem work with parsed function parameters
closes #29744

GiudGiud committed:
Add docstring to functor DirichletBC
Use AD functor dirichlet BC for temperature for coupling purposes

GiudGiud committed:
Avoid forgetting to use block restriciton in heat conduction physics
refs #29746

GiudGiud committed:
Add pressure and lambda to the system for the flow physics
Lower thermal expansion
Add scaling to energy equation to accomodate for bad convergence

GiudGiud committed:
Use flux BCs in water, reduce cp to obtain T profile faster Add postprocessing Precondition each system separately

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2647681/), step Clang format on 9b1393c wanted to post the following:

Your code requires style changes.  

A patch was auto generated and copied [here](https://mooseframework.inl.gov/docs/PRs/27572/clang_format/style.patch)  
You can directly apply the patch by running, in the top level of your repository:

`curl -s https://mooseframework.inl.gov/docs/PRs/27572/clang_format/style.patch | git apply -v`  

Alternatively, with your repository up to date and in the top level of your repository:

`git clang-format bf7a391b5da0313b05c3181a855130d329ea5b12`

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2647779/), step Fixup commit check on 78b70d8 wanted to post the following:

The following commit(s) contain fixups:

- 4fa86f0721: fixup! Use flux BCs in water, reduce cp to obtain T profile faster Add postprocessing Precondition each system separately

GiudGiud committed:
Add a homogenized aluminum layer into the concrete

GiudGiud committed:
Add doco for step 12

GiudGiud committed:
Adapt to new executable location (if using local executable)

GiudGiud committed:
Add support for multi-system to dumpobjectsProblem
closes #29743

GiudGiud committed:
Remove FE NS flow for step 7
Use FV
Show FV flow in 3D then do the same transition to 2D for investigating the coupling

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit 8dc9ab6699d0c9c5d9ae03665a2ece62eb339b49
+Subproject commit 3ba8fc30775a481abc3663171da785ce47b39bfd
```
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

GiudGiud committed:
Use a nearest node auxkernel to get the functor material matching to work

GiudGiud committed:
Make NNValueAux a lot safer
- check for nodal variables
- dont seg fault on non-nodal ones
- add support for non-nodal variables
- add more checks
closes #29741

GiudGiud committed:
Remove all C++ from systems definitions
Add more on some systems

zachmprince committed:
New tutorial mesh, who dis?

zachmprince committed:
Working through coupled fluid problem

zachmprince committed:
Lots of changes to inputs, almost finished I think

zachmprince committed:
Rearranging tutorial steps a bit

zachmprince committed:
Starting solid-fluid heat transfer coupling via multiapps in tutorial

zachmprince committed:
Finalizing multiphysics inputs

zachmprince committed:
Final modifications to inputs based on workshop narrative

zachmprince committed:
Overhaul of user training based on new model and getting multiphysics working

zachmprince committed:
Finalize testing for user tutorial

zachmprince committed:
Getting the link to the NS slides working

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit 8dc9ab6699d0c9c5d9ae03665a2ece62eb339b49
+Subproject commit 7df25da9663f229a3a89db8e8e3bfe66caf9f712
```
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

zachmprince committed:
Adding all the tutorial inputs into their own directory for easy test install

zachmprince committed:
Adding tutorial to copyable inputs

zachmprince committed:
Fixing parallel issue in step10 and adding singular value results to slides

zachmprince committed:
Suggestion 1: Introducing physics being solved early before getting into governing equations

zachmprince committed:
Suggestion 2: Removing mention of 'not coding'

zachmprince committed:
Suggestion 3: Introducing the mesh more gradually for comprehension

zachmprince committed:
Suggestion 4: Removing all FEProblem in the inputs

zachmprince committed:
Suggestion 5: Removing mention of AD that's too early

zachmprince committed:
Suggestion 6: Adding links to object pages when they are mentioned

zachmprince committed:
Suggestion 7: Adding visualization of sidesets

zachmprince committed:
Suggestion 8: Removing 'Testing' and 'MMS' sections

zachmprince committed:
Suggestion 9: Making multiapp step clearer on the difference in fluid and sensor coupling, as well as removing the heat flux dirac kernel

zachmprince committed:
Making fluid physics more realistic and adding turbulence modeling

zachmprince committed:
Updating link name for workshops

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit 7df25da9663f229a3a89db8e8e3bfe66caf9f712
+Subproject commit af3ee20e55f7e493d807db706e473da5a7e6027d
```
**Caution!** This contains a submodule update

zachmprince commented:
> @zachmprince what were we missing here?

@GiudGiud nothing from my end, it is ready for review.

GiudGiud commented:
Test failure is unrelated.
We are missing testing. @loganharbour could you please help with that? We would ideally use the conda all-inclusive binary to run the tests since that's how the tutorial is meant to be ran.

Alternatively I can set something up with combined but it's less ideal

GiudGiud commented:
step12 results are funky because of gravity. I think it's fixable, I ll push another commit.

GiudGiud committed:
Add more refinement to final case (runs on an M4 still)
Add a symmetry BC to avoid gravity slump on right side

GiudGiud committed:
Fix a couple formatting issues:
- too big for page
- extra empty slides

GiudGiud committed:
Loosen tolerance a tad on physics solve (could also let it cut the step)
Limit testing length to 1000s in simulation time

GiudGiud commented:
Testing works :)

licharlot commented on tutorials/shield_multiphysics/doc/content/user_workshop/index.md:
```diff
@@ -0,0 +1,286 @@
+
+## (continued)
+
+!include user_workshop/problem/step12.md start=end-intro
```
This step is mixing the custom syntax and multi-system. Could be split into 2 steps.

GiudGiud replied:
This is how I intend Physics to be used and the case is not big enough to show that fully coupled is bad

licharlot commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step01.md:
```diff
@@ -0,0 +1,185 @@
+!equation
+-\nabla \cdot \nabla T = 0 \in \Omega,
+
+where $T = 300$ on the back ($z=0$), $T = 330$ on the front ($z=5.25$) and with
```
The name of the front and back sidesets is not intuitive since that are on top and bottom

licharlot commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step01.md:
```diff
@@ -0,0 +1,185 @@
+\left(\nabla \psi_i, \nabla T_h\right) = 0 \quad \forall \psi_i,
+
+where $\psi_i$ are the test functions and $T_h$ is the finite element solution.
+
```
Not sure the weak form is necessary here. Would be more meaningful after the fem intro

GiudGiud replied:
you're right. deleting

licharlot commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step01.md:
````diff
@@ -0,0 +1,185 @@
+
+```bash
+cd ~/projects/moose/tutorials/shield_multiphysics/inputs/step01_diffusion
+moose-opt -i mesh.i --mesh-only
````
Put this execution step after the geometry definition?

licharlot commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step01.md:
````diff
@@ -0,0 +1,185 @@
+moose-opt -i mesh.i --mesh-only
+moose-opt -i step1.i
+```
+
````
Would be nice to have a description of the info in the console output (number of DoF, etc.)

GiudGiud replied:
adding output

licharlot commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step02.md:
```diff
@@ -0,0 +1,56 @@
+!!end-intro
+
+!---
+
```
In the presentation, there is a lot of content between these 2 slides. Could be nice to have a reminder. Also suggest summarizing the values of the diffusion coefficient on each block, maybe with a picture?

GiudGiud replied:
Ok added a reminder.

licharlot commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step02.md:
```diff
@@ -0,0 +1,56 @@
+
+## Step 2: Result
+
+!media shield_multiphysics/results/step02.png
```
the T fluid and T Solid labels are confusing since there is only one variable in the input file.

GiudGiud replied:
This has to be the wrong picture

GiudGiud replied:
Deleted one colorbar so only one shows

GiudGiud replied:
@zachmprince could you please re-generate the picture? The current one does not match any step I think

GiudGiud replied:
We can fix this in large_media btw, we dont need to change anything in this PR to fix the image

licharlot commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step03.md:
```diff
@@ -0,0 +1,91 @@
+- natural convection boundary conditions with the air around the shield
+
+- convective boundary conditions with the water block
+
```
Would be nice to add a figure illustrating the BCs

GiudGiud replied:
done

licharlot commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step03.md:
```diff
@@ -0,0 +1,91 @@
+
+!col! width=50%
+
+!media shield_multiphysics/results/step03_mesh.png style=width:100%;margin-left:auto;margin-right:auto;display:block
```
Make sure to explain `water_boundary` vs `water_boundary_inward`

licharlot commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step03.md:
```diff
@@ -0,0 +1,91 @@
+
+Convection with water
+
+We use the same boundary condition as for with for now. When we introduce a separate variable for the
```
Please correct  this sentence.

GiudGiud replied:
fixed

licharlot commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step03.md:
```diff
@@ -0,0 +1,91 @@
+
+## Step 3: Result
+
+!media shield_multiphysics/results/step03.png
```
Need to explain that we don't solve on the water block.

GiudGiud replied:
done

licharlot commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step03.md:
```diff
@@ -0,0 +1,91 @@
+## Step 3: Result
+
+!media shield_multiphysics/results/step03.png
+
```
Need to explain why some temperatures are below 300K

GiudGiud replied:
I have no such temperatures running this locally. I think it's just the paraview colorbar

licharlot reviewed (commented):
Very nice tutorial. Some of the theoretical sections might be a bit too much for a user, but it is good to have. 
I'd suggest adding a section on the input file syntax (commenting lines, unit conversion, global parameters, etc)

GiudGiud committed:
Address Lise's review:
Add a section on input file syntax

GiudGiud commented on tutorials/shield_multiphysics/doc/content/user_workshop/index.md:
```diff
@@ -0,0 +1,286 @@
+
+!---
+
+!include user_workshop/systems_user/functions.md
```
moved functions with BCs. The functions description uses BCs

GiudGiud commented on tutorials/shield_multiphysics/doc/content/user_workshop/systems_user/timesteppers.md:
```diff
@@ -0,0 +1,61 @@
+#  [Time Stepper System](Executioner/TimeSteppers/index.md)
+
+A system for suggesting time steps for transient executioners.
```
switched to computing

GiudGiud commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step04.md:
```diff
@@ -0,0 +1,61 @@
+## HeatConduction Kernel
+
+The [CoefDiffusion](CoefDiffusion.md) Kernel object uses input parameters for defining the thermal
+conductivity. Instead, the [HeatConduction](HeatConduction.md) Kernel utilizes the material
```
I ll add a brief note. We explain AD in the fem section so this should be fine to explain

GiudGiud commented on tutorials/shield_multiphysics/inputs/step07_mechanics/step7.i:
```diff
@@ -0,0 +1,206 @@
+    block = 'Al'
+  []
+
+  # NOTE: This handles thermal expansion by coupling to the displacements
```
unforturnately not. The thermal expansion is really handled by the density material property. There is no kernel for it

GiudGiud commented on tutorials/shield_multiphysics/doc/content/user_workshop/problem/step10.md:
```diff
@@ -0,0 +1,92 @@
+
+!equation id=fluid_velocity
+\frac{\partial \rho  \vec{u}}{\partial t} + \nabla \cdot \left(\rho \vec{u} \otimes \vec{u}\right)
+= \nabla \cdot \left(\mu_\text{eff} \left(\nabla\vec{u}_I + \left(\nabla \vec{u}_I\right)^T-\frac{2}{3}\nabla\cdot\vec{u}_I\mathbb{I}\right)\right) -\nabla p + \rho \vec{g}
```
copy-paste error. deleting _I for intersititial

GiudGiud commented on tutorials/shield_multiphysics/inputs/step11_multiapps/step11_2d_heat_conduction.i:
```diff
@@ -0,0 +1,226 @@
+
+[AuxVariables]
+  [T_fluid]
+    # family = MONOMIAL
+    # order = CONSTANT
```
deleted

GiudGiud committed:
Address Lise's review:
Rename boundaries to make more sense

GiudGiud committed:
Address Lise's review:
Fix test requirement
Remove unused comment
Re-add T_fluid as an aux-variable

GiudGiud committed:
Address Lise's review:
Present mesh & input parameters sections earlier, and functions with BCs

GiudGiud committed:
Address Lise's review:
Add simulation logs
Reminder of coefdiffuion
Add missing words
Add misising dot
Add meshing commands to relevant steps

GiudGiud committed:
Update large_media

GiudGiud commented:
All comments should be addressed, I just need to check that the slides still build as expected

GiudGiud committed:
Add graph extension to build plots from Darcy tutorial

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit 9bc8c929b3d21601b4d73b4fb2eb02ed06aba657
+Subproject commit 86e24765716fc8d8bf77318877af58166d8258b2
```
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

GiudGiud committed:
Avoid file name conflict
Give link to main doco page on input_syntax

GiudGiud committed:
Try a re-org of content

GiudGiud commented:
Looks fine. Tests are fixed now

GiudGiud commented:
Test failure unrelated

licharlot reviewed (commented):
I think it flows much better now! Just a few comestic details to fix

GiudGiud committed:
Use correct background slide
Re-size mesh in step 10

GiudGiud committed:
Update large_media

moosebuild commented on large_media:
```diff
@@ -1 +1 @@
-Subproject commit 9bc8c929b3d21601b4d73b4fb2eb02ed06aba657
+Subproject commit 9332e29706d1a0413b307d0d746096f6e790615a
```
**Caution!** This contains a submodule update

GiudGiud commented:
Thanks for the review @licharlot !
@loganharbour if you want to take another look

GiudGiud commented:
libtorch cuda and python 3.9 failures are from recipe changes (no longer exist for 3.9)
